intel-retail/automated-self-checkout
# feat: parameterize batch-size in gst pipelines via environment variable

Use new env var BATCH_SIZE to parameterize batch-size element in gst pipelines, specifically gvadetect command

CLOSES: #29

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**29**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- re-build profile-launcher: `make build-profile-launcher`
- re-build gst profile docker image: `make build-dlstreamer`
- start camera-simulator: `make run-camera-simulator`
- you can run gst pipeline without specifying BATCH_SIZE: `PIPELINE_PROFILE=gst RENDER_MODE=0 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0`   the default value of BATCH_SIZE is 0; or run with specifying the BATCH_SIZE env like `PIPELINE_PROFILE=gst RENDER_MODE=0 BATCH_SIZE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0`
- smoke tests should ALL PASSED: `make run-smoke-tests`


## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
